### ISSUE TITLE
async optional transformer + super -> Cannot read property 'body' of undefined

### ISSUE THREAD

**Issue by mpal9000** [6to5/6to5#580]
Hello, 6to5 rocks! Here is an issue:

```javascript
function Base() {}

Base.prototype.meth = async function(arg) {
  return arg;
};

class Extended extends Base {
  async meth(...args) {
    return await super.meth(...args);
  }
}
```

```shell
# this works
6to5 --experimental ./6to5-test.js

# TypeError: ./6to5-test.js: Cannot read property 'body' of undefined
6to5 --optional=asyncToGenerator ./6to5-test.js
6to5 --experimental --optional=asyncToGenerator ./6to5-test.js
6to5 --optional=bluebirdCoroutines ./6to5-test.js
6to5 --experimental --optional=bluebirdCoroutines ./6to5-test.js
```

**Comment by sebmck** [comment]
So this is a potentially tricky issue to solve because the `classes` transformer needs to be ran after the `modules` transformer but the `asyncToGenerator` transformer needs to be ran after the `classes` transformer but before the `modules` transformer...

**Comment by mpal9000** [comment]
Maybe the optional transformer can be split up or run twice...

**Comment by sebmck** [comment]
Fixed in the 3.0.0 branch

**Comment by mpal9000** [comment]
Thank you

**Comment by sebmck** [comment]
Fixed as of 3.0.0
